todo_app/flask_config.py

```diff
@@ -3,11 +3,6 @@
 class Config:
     def __init__(self):
         """Base configuration variables."""
-        self.SECRET_KEY = os.environ.get('SECRET_KEY')
```

Comment:
Nice cleaning here - we still have a reference to that variable in .env.template which we could remove too 😄